tests/runtime/dbk_utils.hh

```diff
@@ -0,0 +1,348 @@
+/**
+ * \brief OpenCL runtime library: poclu - useful C++ utility functions for
```

Comment:
Is this accurate? Seems to be utils for testing DBKs?

Reply:
Addressed (was stale comment).